azurerm/resource_arm_service_fabric_cluster.go

```diff
@@ -372,6 +381,7 @@ func resourceArmServiceFabricClusterUpdate(d *schema.ResourceData, meta interfac
 			NodeTypes:                    nodeTypes,
 			ReliabilityLevel:             servicefabric.ReliabilityLevel1(reliabilityLevel),
 			UpgradeMode:                  servicefabric.UpgradeMode1(upgradeMode),
+			ClusterCodeVersion:           utils.String(clusterCodeVersion),
```

Comment:
we've run the test suite here and this fails because the `ClusterCodeVersion` is an empty string when it's not specified (so this fails on the API Validation):

```
* azurerm_service_fabric_cluster.test: Error creating Service Fabric Cluster "acctest-9043244035322992792" (Resource Group "acctestRG-9043244035322992792"): servicefabric.ClustersClient#Create: Failure sending request: StatusCode=0 -- Original Error: autorest/azure: Service returned an error. Status=400 Code="InvalidRequestFormat" Message="Cannot parse the request." Details=[{"code":"InvalidRequestFormat","message":"Error parsing version string:. Path 'properties.clusterCodeVersion', line 1, position 115."}]
```

could we update this to only set the field if it's got a value? e.g.
```
if clusterCodeVersion != "" {
  parameters.ClusterPropertiesUpdateParameters.ClusterCodeVersion = utils.String(clusterCodeVersion)
}
```

(we also need to update the Create above to do the same thing)